site/docs/guide/filter-queries.md

```diff
@@ -319,3 +354,6 @@ These functions are then combined to form a predicate that only has to check for
 
 This system uses less operations than some competing libraries, which need to perform containment checks in arrays when routing updates.
 grammY's filter query system is much more powerful.
+
+```
```

Comment:
Why we have this without any code in it?

Reply:
Fixed.

Reply:
That's an artefact of formatting Markdown with incomplete code environments. The two lines should be removed.